hyundai_kia_connect_api/ApiImplType1.py

```diff
@@ -49,7 +49,7 @@ def _get_authenticated_headers(
     def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None:
         if get_child_value(state, "Date"):
             # `Date` field is in UTC time
-            if vehicle.last_updated_at:
+            if not vehicle.last_updated_at:
                 vehicle.last_updated_at = parse_datetime(
                     get_child_value(state, "Date"), dt.timezone.utc
```

Comment:
I think this was wrong, someone else already reported that this was wrong for ccs2, it should be self.data_timezone